src/containers/StructurePage/folderComponents/SettingsMenuDropdownType.tsx

```diff
@@ -81,6 +83,11 @@ const SettingsMenuDropdownType = ({ rootNodeId, node, onCurrentNodeChanged, node
               })}
             </TabsTrigger>
             <TabsTrigger value="toggleMetadataVisibility">{t("metadata.changeVisibility")}</TabsTrigger>
+            <TabsTrigger value="addProgramme">
+              {t("taxonomy.addNode", {
+                nodeType: t(`taxonomy.nodeType.${nodeType}`),
+              })}
+            </TabsTrigger>
```

Comment:
Må man være taxonomi-admin for å legge til nytt utdanningsprogram og nye emner? @gunnarvelle